calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -56,6 +57,21 @@ def post_usage_async
       private
 
       # @!visibility private
+      def preferences
+        @preferences ||= Calabash::Cucumber::Preferences.new
+      end
+
+      # @!visibility private
+      def user_id
+        @user_id ||= preferences.user_id
+      end
+
+      # @!visibility private
+      def allowed_to_track
+        @allowed_to_track ||= preferences.usage_tracking
+      end
+
+      # @!visibility private
       def self.track_usage?
         @@track_usage && !self.xtc?
```

Comment:
@TobiasRoikjer you added this while we were pairing; do you have a strong argument for keeping it.

> I prefer to avoid having two switches that perform the same function - IMO it leads to some unnecessary complexity.

I think the best argument for keeping this behavior is that it allows users to enable/disable tracking on a per-project basis without having to touch the preferences file.